src/libs/protocols/protos/agent.proto

```diff
@@ -505,3 +507,55 @@ message GetMetricsRequest {}
 message Metrics {
 	string metrics = 1;
 }
+
+message VolumeStatsRequest {
+	string path = 1;
+}
+
+message VolumeStatsResponse {
+	// This field is OPTIONAL.
+	repeated VolumeUsage usage = 1;
+	// Information about the current condition of the volume.
+	// This field is OPTIONAL.
+	// This field MUST be specified if the VOLUME_CONDITION node
+	// capability is supported.
+	VolumeCondition volume_condition = 2;
+}
+message VolumeUsage {
+	enum Unit {
+		UNKNOWN = 0;
```

Comment:
Why is this needed?

Reply:
I'm copying this from the [csi spec](https://github.com/container-storage-interface/spec/blob/master/csi.proto#L1449). Happy to remove it.

Reply:
ack, but see my comments above about the csi spec...